### PR TITLE
added jquery.filtertable and waitMe

### PR DESCRIPTION
**Pull request by totpero**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Review by totpero — COMMENTED** [review]
try to fix bom

**Review by totpero — COMMENTED** [review]
try to fix 2

File: types/jquery.filtertable/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,144 @@
+/// <reference path="../jquery/jquery.d.ts" />
```
Comment (andy-ms) [inline types/jquery.filtertable/index.d.ts]: The header must come first, anything else must come later.
Also, use `/// <reference types="jquery" />` instead.

Comment (totpero) [inline types/jquery.filtertable/index.d.ts]: cool, thanks you very much.  👍

**Comment by andy-ms** [comment]
Both need a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

File: types/waitme/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+			"dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```
Comment (minestarks) [inline types/waitme/tsconfig.json]: Please set `"strictNullChecks": true` for new packages

File: types/jquery.filtertable/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```
Comment (minestarks) [inline types/jquery.filtertable/tsconfig.json]: Please set `"strictNullChecks": true` for new packages

**Review by minestarks — CHANGES_REQUESTED** [review]
I'll merge when the `strictNullChecks`s are updated

File: types/waitme/tsconfig.json (hunk 2 above)
Comment (totpero) [reply types/waitme/tsconfig.json]: I have done, Thanks 👍

File: types/jquery.filtertable/tsconfig.json (hunk 3 above)
Comment (totpero) [reply types/jquery.filtertable/tsconfig.json]: I have done, Thanks 👍

**Review by minestarks — APPROVED** [review]

**Comment by minestarks** [comment]
Thank you!

**Pull request merged**